corehq/apps/app_manager/static/app_manager/js/apps_base.js

```diff
@@ -19,4 +19,16 @@ $(function () {
     if (v2) {
         hqLayout.utils.setIsAppbuilderResizing(true);
     }
+
+    $(document).ajaxComplete(function(e, xhr, options) {
+        if (/edit_form_attr/.test(options.url) ||
```

Comment:
👍 sorry i definitely wrote the first iteration of this because it was the easiest thing to get out the door. the one issue i see with the additional request is that there will likely be a lot of false positives if the request is made here since this triggers after any ajax request is made. essentially we want this to update when someone hits the save button, but it felt really hacky to splice that in there